catalog/core/catalog-core-standardframework/src/test/java/ddf/catalog/impl/operations/RemoteDeleteOperationsTest.java

```diff
@@ -0,0 +1,326 @@
+        frameworkProperties.setPreQuery(new ArrayList<>());
+        frameworkProperties.setPostQuery(new ArrayList<>());
+        frameworkProperties.setPreResource(new ArrayList<>());
+        frameworkProperties.setPostResource(new ArrayList<>());
+        frameworkProperties.setQueryResponsePostProcessor(mock(QueryResponsePostProcessor.class));
+        frameworkProperties.setStorageProviders(Collections.singletonList(storageProvider));
+        frameworkProperties.setMimeTypeMapper(new MimeTypeMapperImpl(Collections.singletonList(
+                mimeTypeResolver)));
+        frameworkProperties.setMimeTypeToTransformerMapper(mimeTypeToTransformerMapper);
+        frameworkProperties.setValidationQueryFactory(new ValidationQueryFactory(new GeotoolsFilterAdapterImpl(),
+                new GeotoolsFilterBuilder()));
+
+        SourceOperations sourceOperations = new SourceOperations(frameworkProperties);
+        OperationsSecuritySupport opsSecurity = new OperationsSecuritySupport();
+        MetacardFactory metacardFactory = new MetacardFactory(mimeTypeToTransformerMapper);
+        OperationsMetacardSupport opsMetacard = new OperationsMetacardSupport(frameworkProperties,
+                metacardFactory);
+
+        QueryOperations queryOperations = new QueryOperations(frameworkProperties,
+                sourceOperations,
+                opsSecurity,
+                opsMetacard);
+
+        OperationsCatalogStoreSupport opsCatStore = new OperationsCatalogStoreSupport(
+                frameworkProperties,
+                sourceOperations);
+
+        deleteOperations = new DeleteOperations(frameworkProperties,
+                queryOperations,
+                sourceOperations,
+                opsSecurity,
+                opsMetacard,
+                opsCatStore);
+
+        // set up deleteRequest
+        MetacardImpl metacard = new MetacardImpl();
+        ArrayList<Metacard> metacardList = new ArrayList<>();
+        metacard.setId("Bob");
+        metacard.setTitle("Bob's Title");
+        metacardList.add(metacard);
+
+        metacard = new MetacardImpl();
+        metacard.setId("Bobbert");
+        metacard.setTitle("Bobbert's Title");
+        metacardList.add(metacard);
+
+        deleteRequest = new DeleteRequestImpl(metacard.getId());
+        deleteResponse = new DeleteResponseImpl(deleteRequest, new HashMap(), metacardList);
+
+        // set up blacklist
+        fanoutTagBlacklist = new ArrayList<>();
+        fanoutTagBlacklist.add("");
+
+        opsSecuritySupport = mock(OperationsSecuritySupport.class);
+
+        opsMetacardSupport = mock(OperationsMetacardSupport.class);
+
+        opsCatStoreSupport = mock(OperationsCatalogStoreSupport.class);
+
+        remoteDeleteOperations = new RemoteDeleteOperations(frameworkProperties,
+                opsMetacardSupport,
+                opsCatStoreSupport);
+
+    }
+
+    @Test
+    public void testIsNotCatalogStoreRequest() throws Exception {
```

Comment:
Good test names include the state being tested and the expected result/behavior. `testIsNotCatalogStoreRequest` says what state is being tested, but it says nothing about what the test is expecting.